projects/admin/src/app/menu/menu-user/menu-user.component.ts

```diff
@@ -45,6 +49,7 @@ export class MenuUserComponent implements OnInit {
     // If the user logout, we delete the local storage
     if (event.getAttribute('id') === 'logout-menu') {
       this._librarySwitchMenuStorageService.removeStorage();
+      this._loanFixedDateService.remove();
     }
```

Comment:
General comment : not for this specific PR.
I don't like this implementation where we define all methods to call on a menu action button.
This action button should only emit an event (something like `logoutEvent$.emit(true)`). So multiple components/services could subscribe to this event to perform specific operation when user logout.

Here it's the "menu" component that must know what operations performs... I don't like this behavior (and it should not compatible with angular16 signals)